orc/src/test/java/org/apache/iceberg/orc/TestOrcDataWriter.java

```diff
@@ -141,7 +137,7 @@ public void testUsingFileIO() throws IOException {
     // local files that would be the LocalFileSystem. To prevent this we use the Proxy classes to
     // use a scheme `dummy` that is not handled. Note that Hadoop 2.7.3 throws IOException
     // while latest Hadoop versions throw UnsupportedFileSystemException (extends IOException)
-    ProxyOutputFile outFile = new ProxyOutputFile(Files.localOutput(temp.newFile()));
+    ProxyOutputFile outFile = new ProxyOutputFile(Files.localOutput(String.valueOf(temp.isFile())));
```

Comment:
this seems wrong. This expects a file and not a "true"/"false" string

Reply:
I changed the variable to go inside to fit the context. `String.valueOf(temp.isFile())` to `temp`